03/app.test.js

```diff
@@ -1 +1,29 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('if value min = 1 and value max = 1', () => {
+    expect(randomNumber(1,1)).toBe(1);
+});
+
+it('if value is not a number', () => {
+    function checkingValues(){
+        const min ="oko";
+        const max ="dom";
+        randomNumber(min,max)
+    }
+    expect(checkingValues).toThrow(Error);
```

Comment:
Warto zdefiniować jaka jest treść błędu, bo nie wiemy czy to jest ten, którego się spodziewamy:
https://jestjs.io/docs/expect#tothrowerror